Firmware/Marlin_main.cpp

```diff
@@ -9839,15 +9839,7 @@ if(0)
 						lcd_update_enable(true);
 */
 						eFilamentAction=FilamentAction::AutoLoad;
-						bFilamentFirstRun=false;
-						if(target_temperature[0]>=EXTRUDE_MINTEMP){
-							bFilamentPreheatState=true;
-//							mFilamentItem(target_temperature[0],target_temperature_bed);
-							menu_submenu(mFilamentItemForce);
```

Comment:
I have not tested this line. Need to verify we actually need this call to `menu_submenu`

Reply:
I dropped this commit